NitroxClient/GameLogic/Spawning/NoneEntitySpawner.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel_Subnautica.Helper;
+using UnityEngine;
+using UWE;
+
+namespace NitroxClient.GameLogic.Spawning
+{
+    public class NoneEntitySpawner : IEntitySpawner
```

Comment:
How many entities are we talking here?  Will spawning 'None' be a large performance hit?

Reply:
the performance hit ive seen so far is relatively minor